src/header.rs

```diff
@@ -1218,3 +1218,8 @@ fn decode_item_from_parts<'a>(
         _ => unreachable!(),
     })
 }
+
+/// Decodes a SCALE-encoded transaction bytes.
+pub fn transaction_decode(mut bytes: &[u8]) -> Result<Vec<u8>, parity_scale_codec::Error> {
```

Comment:
parity_scale_codec is not part of the `wasm-node` deps and i didn't want it to be. Therefore, added this fn temporarily here... a suggestion of a more suitable for this to live is appreciated.